Sources/Networking/Core/DownloadAPIManager.swift

```diff
@@ -0,0 +1,220 @@
+//
+//  DownloadAPIManager.swift
+//  
+//
+//  Created by Matej Molnár on 07.03.2023.
+//
+
+import Foundation
+import Combine
+
+/// Default Download API manager
+open class DownloadAPIManager: NSObject, Retryable {
```

Comment:
I would add a little bit more description here, how it works, not necessary proper documentation, that will be revalidated as separate issue

Reply:
Yeah actually realised the protocol for this is still missing, I'll add that and add the description there.